CHANGES.md

```diff
@@ -1,3 +1,14 @@
+## edge-18.10.4
+* CLI
+  * **New** adds the concept of service profiles to the controller
+* Web UI
+  * **New** Revamped the look of the dashboard by switching component libraries
```

Comment:
How about: Major look and feel update to the Linkerd dashboard by switching component libraries from antd to material-ui